### PR TITLE
BXMSPROD-1265: Conscious language initiative

### PR DESCRIPTION
**Pull request by radtriste**
**PLEASE DO NOT MERGE. THIS WILL BE DONE WHEN `master` will be renamed as `main`**

Related PRs:
- https://github.com/kiegroup/kogito-pipelines/pull/188
- https://github.com/kiegroup/kogito-runtimes/pull/1365
- https://github.com/kiegroup/optaplanner/pull/1372
- https://github.com/kiegroup/optaweb-employee-rostering/pull/660
- https://github.com/kiegroup/optaweb-vehicle-routing/pull/532
- https://github.com/kiegroup/optaplanner-quickstarts/pull/144
- https://github.com/kiegroup/kogito-apps/pull/882
- https://github.com/kiegroup/kogito-examples/pull/729
- https://github.com/kiegroup/kogito-images/pull/516
- https://github.com/kiegroup/kogito-operator/pull/905
- https://github.com/kiegroup/rhpam-kogito-operator/pull/44

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: optaweb-vehicle-routing-frontend/package-lock.json [hunk 1]
```diff
@@ -5,16 +5,12 @@
   "dependencies": {
     "@babel/code-frame": {
       "version": "7.0.0",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
```
Comment (yurloc) [inline optaweb-vehicle-routing-frontend/package-lock.json:8]: Do not modify `package-lock.json`.

Comment (radtriste) [reply optaweb-vehicle-routing-frontend/package-lock.json:8]: that was done by vscode ... gonna revert that change

Comment (radtriste) [reply optaweb-vehicle-routing-frontend/package-lock.json]: @yurloc done

Comment (yurloc) [reply optaweb-vehicle-routing-frontend/package-lock.json]: Perfect! Thanks :slightly_smiling_face:

**Review by yurloc — APPROVED** [review]
LGTM.

**Pull request merged**